pallets/afloat/src/lib.rs

```diff
@@ -255,46 +307,45 @@ pub mod pallet {
 			Ok(())
 		}
 
+
 		#[pallet::call_index(5)]
 		#[pallet::weight(Weight::from_ref_time(10_000) + T::DbWeight::get().reads_writes(1,1))]
-		pub fn accept_offer(
+		pub fn start_take_sell_order(
```

Comment:
It's okay for now, but I prefer the idea of having an extrinsic that receives an order, and then the backend/pallet determines the flow accordingly. For example, we can process it as a sell order if the type is 'sell,' and so on.